test/system/workflow_executions_test.rb

```diff
@@ -342,6 +342,19 @@ class WorkflowExecutionsTest < ApplicationSystemTestCase
     assert_selector 'div.random_seed-param > input[value="1"]'
   end
 
```

Comment:
Can I also get you to add a test for a samplesheet with a fastq_1 and fastq_2 input

Reply:
Added in 8d54e7c